src/main.ts

```diff
@@ -12,6 +13,9 @@ export async function run() {
     // If not supplied then task is still used to setup proxy, auth, etc...
     //
     let version = core.getInput('node-version');
+    if (!version) {
```

Comment:
This will break the option `core.getInput('version')`

Reply:
ah yeah of course. moving the `preferredNodeVersion` to the end should still allow for both options, right?